README.md

```diff
@@ -31,32 +31,29 @@ installed).
 
 The user discussion and development of Git take place on the Git
 mailing list -- everyone is welcome to post bug reports, feature
-requests, comments and patches to git@vger.kernel.org (read
+requests, comments, and patches to git@vger.kernel.org (read
```

Comment:
This Oxford comma is a matter of hot debate, and prone to lead to lengthy discussions of dubious value. It is correct "enough" without the comma, hence I would prefer for it to be left alone.